arch/WASM/WASMInstPrinter.c

```diff
@@ -0,0 +1,57 @@
+/* Capstone Disassembly Engine */
+/* By Spike, xwings  2019 */
+
+#include "WASMInstPrinter.h"
+#include "WASMMapping.h"
+
+
+void WASM_printInst(MCInst *MI, struct SStream *O, void *PrinterInfo)
+{
+	int i;
+
+	SStream_concat(O, WASM_insn_name((csh)MI->csh, MI->Opcode));
+
+	switch (MI->wasm_data.type) {
+		default:
```

Comment:
Is placing the default case at the beginning also part of the capstone coding style? didn't saw that before

Reply:
i am not sure if Linux kernel coding style follows this, but personally i find that having default case at the top is easier to track.